src/dialog-editor/services/modal/modalService.ts

```diff
@@ -98,7 +98,9 @@ class ModalController {
       attributes: ['description', 'single_value', 'children'],
     };
```

Comment:
the query method is not available here?

if so then you don't need the `options` do you?